### PR TITLE
Fix for Issue #21512 and code cleanup for Jakefile.js

### PR DESCRIPTION
**Pull request by TravCav**
Mostly to fix Issue #21512

Jakefile.js was throwing lint warnings so I cleaned some of them up.

### CODE REVIEW

File: Jakefile.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 // This file contains the build logic for the public repo
 // @ts-check
+/*jshint esversion: 6 */
```
Comment (mhegazy) [inline Jakefile.js]: can we not include this line?

Comment (TravCav) [reply Jakefile.js]: Can we include a .jshintrc file?

**Comment by mhegazy** [comment]
thanks!

**Comment by amcasey** [comment]
This change seems to prevent `jake lint` from reporting any lint errors.  Here's a run followed by an invocation of the command it prints out:
```
e:\ts_gh>jake lint
Linting: node node_modules/tslint/bin/tslint Gulpfile.ts scripts/generateLocalizedDiagnosticMessages.ts "scripts/tslint/**/*.ts" "src/**/*.ts" --exclude "src/lib/*.d.ts" --formatters-dir ./built/local/tslint/formatters --format autolinkableStylish

e:\ts_gh>node node_modules/tslint/bin/tslint Gulpfile.ts scripts/generateLocalizedDiagnosticMessages.ts "scripts/tslint/**/*.ts" "src/**/*.ts" --exclude "src/lib/*.d.ts" --formatters-dir ./built/local/tslint/formatters --format autolinkableStylish

src/services/utilities.ts
ERROR: src/services/utilities.ts:1375:51 semicolon Missing semicolon
```

**Pull request merged**